File: tv/src/main/java/com/synergics/stb/iptv/leanback/models/db/MyDatabase.java
package com.synergics.stb.iptv.leanback.models.db;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.synergics.stb.iptv.leanback.models.TVCategories;
import com.synergics.stb.iptv.leanback.models.TVItems;

@Database(entities = {TVCategories.class, TVItems.class}, version = 4, exportSchema = false)
public abstract class MyDatabase extends RoomDatabase {
    public abstract TVItemDao tvItemDao();
    public abstract TVCategoriesDao tvCategoriesDao();
}

File: tv/src/main/java/com/synergics/stb/iptv/leanback/models/db/MyDatabase.kt
package com.synergics.stb.iptv.leanback.models.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.synergics.stb.iptv.leanback.models.TVCategories
import com.synergics.stb.iptv.leanback.models.TVItems

@Database(entities = [TVCategories::class, TVItems::class], version = 4, exportSchema = false)
abstract class MyDatabase : RoomDatabase() {
    abstract fun tvItemDao(): TVItemDao
    abstract fun tvCategoriesDao(): TVCategoriesDao
}